src/modules/blocks/rsvp/inactive-block/template.js

```diff
@@ -9,28 +9,46 @@ import PropTypes from 'prop-types';
  */
 import { __ } from '@wordpress/i18n';
 
-/**
- * Internal dependencies
- */
-import InactiveBlock, { LAYOUT } from '@moderntribe/tickets/elements/inactive-block';
-import { RSVPInactive } from '@moderntribe/tickets/icons';
-
-const inactiveBlockProps = {
-	className: 'tribe-editor__rsvp__inactive-block',
-	icon: <RSVPInactive />,
-	layout: LAYOUT.rsvp,
-};
-
-const RSVPInactiveBlock = ( { created } ) => {
-	inactiveBlockProps.title = created
-		? __( 'RSVP is not currently active', 'event-tickets' )
-		: __( 'There is no RSVP configured', 'event-tickets' );
-
-	inactiveBlockProps.description = created
-		? __( 'Edit this block to change RSVP settings.', 'event-tickets' )
-		: __( 'Edit this block to create an RSVP form.', 'event-tickets' );
-
-	return <InactiveBlock { ...inactiveBlockProps } />;
+const RSVPInactiveBlock = ({ created }) => {
+	const title = created
+		? __('RSVP is not currently active', 'event-tickets')
+		: __('Add an RSVP', 'event-tickets');
+
+	const description = created
+		? __('Edit this block to change RSVP settings.', 'event-tickets')
+		: __('Allow users to confirm their attendance.', 'event-tickets');
+
+	return (
+		<div class="tribe-common tribe-editor__inactive-block--rsvp">
+			<div class="tribe-editor__inactive-block--rsvp__container">
+				<div class="tribe-editor__rsvp tribe-common-g-row tribe-common-g-row--gutters">
+					<div class="tribe-editor__rsvp-details-wrapper tribe-common-g-col">
+						<div class="tribe-editor__rsvp-details">
+							<h3 class="tribe-editor__rsvp-title tribe-common-h2 tribe-common-h4--min-medium">
+								{title}
+							</h3>
+
+							<div class="tribe-editor__rsvp-description tribe-common-h6 tribe-common-h--alt tribe-common-b3--min-medium">
+								{description}
+							</div>
+						</div>
+					</div>
+
+					<div class="tribe-editor__rsvp-actions-wrapper tribe-common-g-col">
+						<div class="tribe-editor__rsvp-actions">
+							<div class="tribe-editor__rsvp-actions-rsvp">
+								<div class="tribe-editor__rsvp-actions-rsvp-create">
+									<button class="tribe-common-c-btn tribe-common-b1 tribe-common-b2--min-medium">
+										{ __('Create RSVP', 'event-tickets')}
+									</button>
+								</div>
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	);
```

Comment:
Before we had a `<InactiveBlock>` should we follow that same kind of pattern? Maybe, not sure tho.

Reply:
Reading again my previous comment, I feel there is no need to create a reusable component, since only RSVP is going to use this style @bordoni

Reply:
Sounds good.